components/organisms/RepositoriesTable/repositories-table.tsx

```diff
@@ -82,7 +87,7 @@ const RepositoriesTable = ({
 
       {isLoadedWithRepos &&
         listOfRepositories.map((item, index) => (
-          <RepoRow key={`${item.handle}/${item.name}/${index}`} topic={topic} repo={item} user={user} />
+          <RepoRow key={`${item.handle}/${item.name}/${index}`} topic={topic} repo={item} user={user} selected={selectedRepos.find((r) => r.id == item.id) != undefined} handleOnSelectRepo={handleOnSelectRepo} />
```

Comment:
this is kind of mismatching the fact that the type is set to array but we are transporting undefined, why not force the variable to be an empty array instead of doing the filtering here?

Reply:
sorry, type of what exactly? which variable are you talking about?

Reply:
repo can be an array of undefined repos

Reply:
if you mean the parameter "repo", that takes a single repo object and is always defined.

I think you mean the "selected" parameter. Well the line `selectedRepos.find((r) => r.id == item.id) != undefined` evaluates to a boolean, so there isn't any arrays being passed here.
Excuse me if that's not what you meant.